pkg/loader/openapi.go

```diff
@@ -15,15 +16,26 @@ import (
 // Each operation will become a tool definition.
 // The tool's Instructions will be in the format "#!sys.openapi '{JSON Instructions}'",
 // where the JSON Instructions are a JSON-serialized engine.OpenAPIInstructions struct.
-func getOpenAPITools(t *openapi3.T) ([]types.Tool, error) {
+func getOpenAPITools(t *openapi3.T, defaultHost string) ([]types.Tool, error) {
+	// Determine the default server.
 	if len(t.Servers) == 0 {
-		return nil, fmt.Errorf("no servers found in OpenAPI spec")
+		if defaultHost != "" {
+			u, err := url.Parse(defaultHost)
+			if err != nil {
+				return nil, fmt.Errorf("invalid default host URL: %w", err)
+			}
+			u.Path = "/"
+			t.Servers = []*openapi3.Server{{URL: u.String()}}
+		} else {
+			return nil, fmt.Errorf("no servers found in OpenAPI spec")
+		}
```

Comment:
This adds support for a "default host" if no server is specified at the global level in the OpenAPI document.